helpers/focus.js

```diff
@@ -179,3 +179,12 @@ export function isFocusable(node, includeHidden, includeTabbablesOnly, includeDi
 	return _isFocusable;
 
 }
+
+export function tryApplyFocus(elem) {
+	const focusable = findComposedAncestor(elem, (node) => (isFocusable(node) || getFirstFocusableDescendant(node) !== null));
+	if (focusable) {
+		forceFocusVisible(focusable);
```

Comment:
I _think_ we always want to force visible focus here -- at least we do from the dialog case... so maybe this is fine. If not, we could always make it an option in the future.

Reply:
I believe that focusable can never be null here. If we traverse up the DOM, we eventually hit the body, which is focusable (but doesn't help much). Effectively, we will force visible focus in every scenario, and return false only if something very weird happens...